docs/technical_reference_guide/tables/options/HVAC Heating Efficiency - heat_pump.tex

```diff
@@ -1,9 +1,9 @@
-\begin{customLongTable}{ |p{3cm}|p{2.25cm}|p{2.25cm}|p{2.25cm}|p{2.25cm}|p{2.25cm}| }
+\begin{customLongTable}{ |p{3cm}|p{2.75cm}|p{2.75cm}|p{2.75cm}|p{2.75cm}|p{2.75cm}|p{2.75cm}|p{2.75cm}| }
```

Comment:
There is a formatting problem here. First of all, I hardcode in the `create_characteristics_tex.rb` script to look for option name "ASHP, SEER 10, 6.2 HSPF"; if it finds it, column width is adjusted to 2.25cm to accommodate this extra wide table. However, we have a name change to "ASHP, SEER 10, 6.8 HSPF" in this PR. But, even if I fix this name, this table is too wide to fit on the page (since we're adding additional efficiency type columns here). I'm not immediately sure what to do here.